### PR TITLE
New Python SimpleFRAP.ipynb

### PR DESCRIPTION
**Pull request by will-moore**
Adding new example notebooks.

### CODE REVIEW

File: Python/SimpleFRAP.ipynb [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from omero.gateway import BlitzGateway\n",
+    "from getpass import getpass\n",
+    "\n",
+    "HOST = 'merge-ci-devspace.openmicroscopy.org'\n",
```
Comment (jburel) [inline Python/SimpleFRAP.ipynb]: to replace by outreach.o.org

File: Python/SimpleFRAP.ipynb [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "dataset_id = 5220\n",
+    "channel_label = \"GFP\""
```
Comment (jburel) [inline Python/SimpleFRAP.ipynb:79]: Add option to enter dataset and channel name

File: Python/SimpleFRAP.ipynb [hunk 3]
```diff
@@ -0,0 +1,268 @@
...
+    "    images.append(image)\n",
+    "# Only use the first 2 images\n",
+    "images = images[:2]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Define various methods we need"
```
Comment (jburel) [inline Python/SimpleFRAP.ipynb]: Explain the methods

**Comment by dominikl** [comment]
Like JM already mentioned, server address should be outreach instead of merge-ci-devspace.
And on outreach the GFP channel name is `528.0`. Apart from that the notebook itself works very well! 👍

**Pull request merged**